cou/steps/plan.py

```diff
@@ -594,9 +594,18 @@ async def _get_upgradable_hypervisors_machines(
     )
 
     if cli_force:
+        logger.info("Selected all hypervisors: %s", stringify_units(nova_compute_units))
         return nova_compute_machines
 
-    return await get_empty_hypervisors(nova_compute_units, analysis_result.model)
+    empty_units, empty_machines = await get_empty_hypervisors(
+        nova_compute_units, analysis_result.model
+    )
+    skipped_units = set(nova_compute_units) - set(empty_units)
+
+    logger.info("Found non-empty hypervisors: %s", stringify_units(skipped_units))
+    logger.info("Selected hypervisors: %s", stringify_units(empty_units))
```

Comment:
not-blocking: I really do not see much benefit not doing these logs directly in `get_empty_hypervisors`.